### ISSUE TITLE
Documentation errors for FFTW module in genstdlib.jl

### ISSUE THREAD

**Issue by jakebolewski** [JuliaLang/julia#13285]
As a follow up to #13274 now that #13281 is merged, the FFTW methods are not being documented correctly to be picked up by genstdlib.jl.

cc @nkottary

**Comment by tkelman** [comment]
how do we fix it? I think I also saw a warning about a signature change in eigs

**Comment by MichaelHatherly** [comment]
I was able to fix the major errors/warnings with this diff, https://gist.github.com/MichaelHatherly/b45a571854387a9700e9, but managed to introduce some weird character changes in the process... anyone seen those before?

The `eigs` signature had ended up out of sync, and I needed to add the ``DFT`` and ``FFTW`` modules to `genstdlib.jl` manually (see the `add_all_docs_mod(...)` calls) to get it to recognise the docstrings that got moved inline recently into those modules. Would be good to have that done automatically though.

Also, combining the ``ifftshift`` docstrings into a single one since before both the second docstring was just redocumenting the same ``Function`` rather than a particular ``Method``.
